### PR TITLE
OCPBUGS-23288: [release-4.12] Submodule update 11/16

### PR DESCRIPTION
**Pull request by sebsoto**

### CODE REVIEW

**Comment by sebsoto** [comment]
Dropping
[[vendor] Use v1.25.14+a52e8df k8s and OpenShift libraries](https://github.com/openshift/windows-machine-config-operator/pull/1947/commits/7e5e6f32ee3c5dbe8b05d2d03562a84ea1582260) as it is causing a build error

**Comment by aravindhp** [comment]
> Dropping [[vendor] Use v1.25.14+a52e8df k8s and OpenShift libraries](https://github.com/openshift/windows-machine-config-operator/pull/1947/commits/7e5e6f32ee3c5dbe8b05d2d03562a84ea1582260) as it is causing a build error

@sebsoto is that right move? Shouldn't you be fixing the build?

**Comment by sebsoto** [comment]
@aravindhp I dropped it considering that the goal of this PR is to pick up the kubelet update.

**Comment by aravindhp** [comment]
@sebsoto then please call that out and have a follow up Jira ticket to tackle this build failure.

**Review by aravindhp — CHANGES_REQUESTED** [review]

File: go.mod [hunk 1]
```diff
@@ -27,7 +27,7 @@ require (
 	golang.org/x/sys v0.13.0
 	k8s.io/api v0.26.11
 	k8s.io/apimachinery v0.26.11
-	k8s.io/client-go v0.26.11
+	k8s.io/client-go v0.26.0-alpha.0
```
[the remark this answers is not in the document]
Comment (mansikulkarni96) [reply go.mod:30]: +1

**Comment by aravindhp** [comment]
> /retest-required
> 
> looks like `remaining-required` is not happy here.

@jmguzik any idea what is going on here?

**Comment by aravindhp** [comment]
Oh wait, did we enable the 2-staged pipelines for release-4.12?

**Comment by sebsoto** [comment]
/override ci/prow/nutanix-e2e-operator ci/prow/vsphere-proxy-e2e-operator

**Pull request merged**